client/src/lib.rs

```diff
@@ -457,10 +457,15 @@ impl<'a> RequestBuilder<'a> {
                 recent_hash,
             )
         };
+        match send_config {
+            Some(config) => rpc_client
+                .send_transaction_with_config(&tx, config)
+                .map_err(Into::into),
+            None => rpc_client
+                .send_and_confirm_transaction(&tx)
```

Comment:
Instead of adding an option, should we add a separate `send_with_config` API to be consistent with the solana client?

Reply:
makes sense, i pushed that change to https://github.com/project-serum/anchor/pull/768